CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Add `SentryFlutter.nativeCrash()` using MethodChannels for Android and iOS ([#2239](https://github.com/getsentry/sentry-dart/pull/2239))
```

Comment:
```suggestion
- Add `SentryFlutter.nativeCrash()` using MethodChannels for Android and iOS ([#2239](https://github.com/getsentry/sentry-dart/pull/2239))
  - This can be used to test if native crash reporting works  
```

or something similar, you can redefine the sentence if you want